shell/browser/api/electron_api_app.cc

```diff
@@ -497,14 +497,16 @@ void OnClientCertificateSelected(
 }
 
 #if defined(USE_NSS_CERTS)
-int ImportIntoCertStore(CertificateManagerModel* model,
-                        const base::DictionaryValue& options) {
-  std::string file_data, cert_path;
-  base::string16 password;
+int ImportIntoCertStore(CertificateManagerModel* model, base::Value options) {
+  std::string file_data;
   net::ScopedCERTCertificateList imported_certs;
   int rv = -1;
-  options.GetString("certificate", &cert_path);
-  options.GetString("password", &password);
+
+  std::string* cert_path_ptr = options.FindStringPath("certificate");
+  std::string cert_path = *cert_path_ptr;
```

Comment:
if `certificate` isn't present in the dictionary, `cert_path_ptr` will be `nullptr` and this will SIGSEGV. We should handle that more gracefully.